test/parallel/test-config-file.js

```diff
@@ -60,6 +60,17 @@ test('should parse boolean flag', async () => {
   strictEqual(result.code, 0);
 });
 
+test('should parse boolean flag defaulted to true', async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--experimental-config-file',
```

Comment:
Very minor nit: we can also expose internals in order to directly verify the value of the option that we're setting. This might help decouple the option behaviour from the test. In this case, we're using the behaviour of `process.emitWarning` to validate that `--no-warnings` has been correctly managed.

Example:
https://github.com/geeksilva97/node/blob/99ef802c0031af7d0e3313cda60b6fac8471ac06/test/parallel/test-config-file.js#L479-L491

Reply:
Sorry, @pmarchini . I had missed this. Yeah, you've got a point, but I took that approach to be more strict